### PR TITLE
AnnaP

### PR DESCRIPTION
**Pull request by annaplaton**
Please check code again. Made some corrections.

### CODE REVIEW

File: src/anna/week_1/OddOrEven.java [hunk 1]
```diff
@@ -23,6 +26,9 @@ public static void oddOrEven(int n) {
         } else {
             System.out.println(n + " is odd number");
         }
+    }
 
+    public static String returnTypeOddOrEven(int num) {
```
Comment (SopioKvantaliani) [inline src/anna/week_1/OddOrEven.java:31]: method returnTypeOddOrEven() looks great!

**Review by SopioKvantaliani — CHANGES_REQUESTED** [review]
check again oddOrEven method. I left single comment inside. 
Method returnTypeOddorEven looks great@

File: src/anna/week_1/OddOrEven.java [hunk 2]
```diff
@@ -15,6 +15,9 @@ public static void main(String[] args) {
         int number = 5;
         oddOrEven(number);
         oddOrEven(6);
+
+        String str = returnTypeOddOrEven(50);
+        System.out.println(str);
     }
 
     public static void oddOrEven(int n) {
```
[the remark this answers is not in the document]
Comment (annaplaton) [reply src/anna/week_1/OddOrEven.java:23]: Thank you. I know)) I decided to leave both here.

File: src/anna/week_1/OddOrEven.java (hunk 1 above)
Comment (annaplaton) [reply src/anna/week_1/OddOrEven.java:31]: Thank you

**Pull request merged**